### DISCUSSION TITLE
AI新科技 四領域當紅

### DISCUSSION THREAD

**Discussion by skyling0299** [1101-GS3073-C/Python-Community-GS3073-C#562] [General]
科技股經歷2022年的動盪修正，仍處於庫存去化階段，法人指出，總體經濟趨緩影響AI、雲端企業的資本支出力道，不過全球數位科技轉型長期趨勢並未改變，巨量資料的運算需求也會帶動雲端服務商營運表現提升，增加投資機會。

有人在玩基金的嗎? 報導說得算是實話嗎?
剛剛機器人跟我說這篇文章原創性不足，所以我打算繼續補幾個字看他能不能被計入
報導中有提到幾個領域，多半都是科技的方面(但我們都知道科技是目前趨勢)

資料來源: https://money.udn.com/money/story/5618/6882075

**Comment by Raywei0214** [comment]
回過神才發現，思考AI新科技的存在意義，已讓我廢寢忘食。這必定是個前衛大膽的想法。本人也是經過了深思熟慮，在每個日日夜夜思考這個問題。

**Comment by larry40305** [comment]
當你搞懂後就會明白了。深入的探討AI新科技，是釐清一切的關鍵。

**Comment by Yilin20021016** [comment]
感覺這一些都是非常難的東西 但之後會變主流的東西
